spring-security-modules/spring-security-legacy-oidc/pom.xml

```diff
@@ -49,6 +49,7 @@
         <spring-security-jwt.version>1.0.9.RELEASE</spring-security-jwt.version>
         <jwks-rsa.version>0.3.0</jwks-rsa.version>
         <spring-boot.version>2.4.7</spring-boot.version>
+        <mockito.version>4.4.0</mockito.version>
```

Comment:
Does 5.11.0 give an error?

Reply:
Yes, this is a quick fix to run the build. Once the build passes we can investigate the code changes required with the new version.

Reply:
OK, I will create a separate task for this module.

Reply:
https://team.baeldung.com/browse/JAVA-33819